src/content/en/updates/2017/03/nic57.md

```diff
@@ -0,0 +1,131 @@
+with media content using the new [Media Session API](https://wicg.github.io/mediasession/).
+
+By providing [metadata](/web/updates/2017/02/media-session#set_metadata) to the
+browser about the content being played, you can create rich lock screen
+messaging that includes information such as title, artist, album name, and
+artwork. You can also listen for and
+[respond to user actions](/web/updates/2017/02/media-session#previous_track_next_track)
+taken on the notification itself, such as seeking or skipping.
+
+If you want to get started quickly, Francois has a great
+[Updates post](/web/updates/2017/02/media-session), or check out the official
+[Chrome media sessions samples repo](https://googlechrome.github.io/samples/media-session/)
+on GitHub. 
+
+
+## And more! {: #andmore }
+
+And there’s lots more!
+
+* There are [some](/web/updates/2017/01/payment-request-updates)
+  [improvements](/web/updates/2017/01/payment-request-updates#paymentmethoddata_supports_basic-card)
+  to the Payment Request API.
+* You can specify the color of the text input cursor with the
+  [`caret-color`](https://www.chromestatus.com/feature/5720917787279360)
+  property.
+* Visual effects such as line color and style can be specified with new 
+  [`text-decoration`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration)
+  properties.
+* The [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
+  [Response class](https://developer.mozilla.org/en-US/docs/Web/API/Response)
+  now supports the
+  [`.redirected`](https://developer.mozilla.org/en-US/docs/Web/API/Response/redirected)
+  attribute to help avoid untrustworthy responses and reduce the risk of open
+  redirectors. 
+* All `-webkit-` prefixed IndexedDB global aliases have been removed, after
+  their deprecation in M38. 
+* And one of my favorites -- new
+  [`padStart`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)
+  and 
+  [`padEnd`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd)
+  formatting methods that simplify string padding when aligning console output
+  or printing numbers with a fixed number of digits.
+
+These are just a few of the changes in Chrome 57 for developers. 
+
+## Shout out to Igalia
+
+Finally, a big shout out to engineers and team at 
+[Igalia](https://www.igalia.com/) for their awesome work on Blink. They
+were instrumental on landing the new CSS Grid and `caret-color` features.
+
+## Stay up to date
+
+
+If you want to stay up to date with Chrome and know what’s coming, be sure to
+[subscribe](https://goo.gl/6FP1a5), follow 
+[@ChromiumDev](//twitter.com/chromiumdev) on Twitter and be sure to check out the
+[videos from the Chrome Dev Summit](https://www.youtube.com/playlist?list=PLNYkxOF6rcIBTs2KPy1E6tIYaWoFcG3uj)
+for a deeper dive into some of the awesome things the Chrome team is working on.
```

Comment:
I realize this is essentially a transcript of a video, but this paragraph has issues. Here's what I'd have done automatically if not for that:
* Line 119: add 'and' before 'follow'.
* LIne 120: end sentence after 'Twitter' and start new sentence with 'and'. (Yes, that's allowed, as long as you know what you're doing.)